pkg/msg_status/tx_status.go

```diff
@@ -16,3 +16,10 @@ const (
 	TxStatusReceived TransactionStatus = "RCVD"
 	TxStatusRejected TransactionStatus = "RJCT"
 )
+
+type StatusReasonCode string
+
+const (
+	// StsReasonCreditorInvalid is the "Creditor account number invalid or missing" status reason code
+	StsReasonCreditorInvalid = "AC03"
+)
```

Comment:
This is already included in the package
https://github.com/moov-io/rtp20022/blob/83b0e95e0f82d6263b62dcedc7c5f06d7d85ff59/pkg/msg_status/errors.go#L66